code/datums/gamemode/factions/bloodcult/bloodcult_rituals.dm

```diff
@@ -358,14 +225,430 @@ var/global/list/cult_altars = list()       // List of cult altars in the world.
+	infected_targets += L
+	if(infected_targets >= targets)
+		return TRUE
+	return FALSE
+
+
+////////////////////////////////////////////////////////////////////
+//																  //
+//						PERSONAL RITUALS						  //
+//																  //
+////////////////////////////////////////////////////////////////////
+
+////////////////////////CONFUSION/////////////////////////////
+
+/datum/bloodcult_ritual/blind_cameras
+	name = "Blind Cameras"
+	desc = "confusion runes and talismans...<br>darken their lenses..."
+
+	ritual_type = "confusion"
+	difficulty = "easy"
+	personal = TRUE
+	reward_achiever = 200
+	reward_faction = 2
+
+	keys = list("confusion_camera")
+
+	var/target_cameras = 3
+
+/datum/bloodcult_ritual/blind_cameras/init_ritual()
+	target_cameras = 3
+
+//Called when a cultist is about to hover the corresponding ritual UI button
+/datum/bloodcult_ritual/blind_cameras/update_desc()
+	desc = "confusion runes and talismans...<br>darken their lenses...<br>[target_cameras] to go..."
+
+/datum/bloodcult_ritual/blind_cameras/key_found(var/extra)
+	target_cameras--
+	if(target_cameras <= 0)
+		return TRUE
+	return FALSE
+
+////////////////////////////////////////////////////////////////
+
+/datum/bloodcult_ritual/confuse_crew
+	name = "Confuse Crew"
+	desc = "confusion runes and talismans...<br>bring their nightmares to life..."
+
+	ritual_type = "confusion"
+	difficulty = "medium"
+	personal = TRUE
+	reward_achiever = 200
+	reward_faction = 2
+
+	keys = list(
+		"confusion_carbon",
+		"confusion_papered",
+		)
+
+/datum/bloodcult_ritual/confuse_crew/key_found(var/mob/living/extra)
+	if (!extra.client)
+		return FALSE
+	return TRUE
+
+////////////////////////HARM CREW MEMBERS/////////////////////////////
+
+/datum/bloodcult_ritual/harm_crew
+	name = "Harm Crew"
+	desc = "wield cult weaponry...<br>spill their blood...<br>sear their skin..."
+
+	ritual_type = "harm"
+	difficulty = "medium"
+	personal = TRUE
+	reward_achiever = 200
+	reward_faction = 2
+
+	keys = list(
+		"attack_tome",
+		"attack_cultblade",
+		"attack_blooddagger",
+		"attack_construct",
+		"attack_shade",
+		"attack_ritualknife",
+		)
+
+	var/targets = 3
+	var/list/hit_targets = list()
+
+/datum/bloodcult_ritual/harm_crew/init_ritual()
+	hit_targets = list()
+
+/datum/bloodcult_ritual/harm_crew/update_desc()
+	desc = "wield cult weaponry...<br>spill their blood...<br>sear their skin...<br>at least [targets - hit_targets] different individuals..."
+
+/datum/bloodcult_ritual/harm_crew/key_found(var/mob/living/L)
+	if (iscultist(L))
+		return FALSE
+	if (L in hit_targets)
+		return FALSE
+	hit_targets += L
+	if(hit_targets >= targets)
+		return TRUE
+	return FALSE
+
+////////////////////////SACRIFICE/////////////////////////////
+
+/datum/bloodcult_ritual/sacrifice_mouse
+	name = "Sacrifice Mouse"
+	desc = "a rodent...<br>an altar...<br>and a proper blade..."
+
+	ritual_type = "sacrifice"
+	difficulty = "easy"
+	personal = TRUE
+	reward_achiever = 200
+	reward_faction = 2
```

Comment:
This should probably be an even lower reward to both the faction and individual than sacrificing a monkey.